### ISSUE TITLE
Add LR Scheduler

### ISSUE THREAD

**Issue by banarutz** [banarutz/BNRNet#6]
scheduler = ReduceLROnPlateau(optimizer, 'min', patience = 4, verbose = True, min_lr = .00000001)
exp_lr_scheduler = lr_scheduler.StepLR(optimizer, step_size=7, gamma=0.1)
